app/models/wordlist_entry.rb

```diff
@@ -3,4 +3,6 @@ class WordlistEntry < ApplicationRecord
   validates :wordlist_id, presence: true
   belongs_to :word
   belongs_to :wordlist
+  has_many :wordlist_entry_categories
```

Comment:
Could this be `has_one :wordlist_entry_category_list`?

Or `has_one :category_list`?
`has_many :categories, through: :category_list, source: :category` ?